### ISSUE TITLE
500 response when taking redis as session store

### ISSUE THREAD

**Issue by ql-hl** [Haufe-Lexware/wicked.haufe.io#89]
The portal service returns 500, when the environment variable "PORTAL_SESSIONSTORE_TYPE" is set to "redis", while the redis service was running correctly (service, port, logging, etc.). 
After setting "PORTAL_SESSIONSTORE_TYPE" to "file", everything works again. 

The issue was detected when running locally by docker-compose.

**Comment by DonMartin76** [comment]
I finally found the root cause for this, and of course it's really stupid: In the default `globals.json`, the Redis port is set to `6739`, correct is `6379`. Updating this in your default configuration should fix this:

```
  "sessionStore": {
    "type": "${PORTAL_SESSIONSTORE_TYPE}",
    "host": "${PORTAL_SESSIONSTORE_HOST}",
    "port": "6379",
    "password": ""
  },
```

SORRY!

**Comment by DonMartin76** [comment]
This is fixed in 0.12.2, but if you updated your configuration using 0.12.1, this will still not be correct.
